woocommerce/class-sv-wc-admin-notice-handler.php

```diff
@@ -254,53 +254,56 @@ public function render_admin_notice_js() {
 		self::$admin_notice_js_rendered = true;
 
 		ob_start();
+
 		?>
+		( function( $ ) {
```

Comment:
@wvega pointed out that it may be best to wrap functions that end up in `wc_enqueue_js()` with an anonymous function call so that any variable set by these snippets in the global namespace do not get accidentally overwritten or overwrite third party code, since everything is funnelled into `wc_enqueue_js()` with a long string. So I've updated our usage here.